### ISSUE TITLE
[Feature Request] Remove downloaded items that failed to auto-import

### ISSUE THREAD

**Issue by hwamil** [ManiMatter/decluttarr#78]
First of all, thank you for this huge QOL you all made possible with `decluttarr`.

Often times, `lidarr` fails to auto-import albums because it cannot match the tracks / wrong tracks/ etc., and gets piled up in the activities queue.

I think it would be a nice option to have these items removed automatically, along with the files. Usually, it's more of a headache than it is worth when I try to import them manually.

Less than `lidarr`, but `radarr` and `sonarr` have fair share of items that fail to auto import and I just do not wish to verify each for manual import anymore...

Thank you for your time in hearing me out!

**Comment by ManiMatter** [comment]
Did you close this request by mistake?

can you share the details of the items you want to remove?

if you go to the network tab of your browser (usually f12) when you are on the queue page, refresh the page and look for the ‚queue‘ entry. 

In the response, please copy the items and paste them here that you want removed

**Comment by ManiMatter** [comment]
Closing since no feedback

**Comment by hwamil** [comment]
@ManiMatter Yes, sorry I closed it on purpose because I felt like I didn't explore it long enough make certain that it does in fact not do this so I wanted to use it for a while first before I make a determination. Apologies for the confusion!

**Comment by ManiMatter** [comment]
Feel free to raise feature suggestions if you feel something could be added to make the tool better
